### DISCUSSION TITLE
WARNING . PYTHON LIBRARY REMOTIOR SENSUS OUTDATED. THIS COULD CAUSE ERROS, PLEASE UP DATE

### DISCUSSION THREAD

**Discussion by macawills** [semiautomaticgit/SemiAutomaticClassificationPlugin#389] [General]
HOW HOW TO UPDATE PYTHON LIBRARY?
MY VERSION OF QGIS: 3.34.13 PRIZEN
                                      3.12.7 PYTHON
                                      3.9.3  GDAL/OGR
SEE THE PRINTS PLEASE IN BB IMAGE. THANKS
   
Hello dear developer community, I'm having a problem with the scp plugin due to the following message:
IT SAYS THAT THE SENSUS REMOTE IS MISSING TO UPDATE, I CLOSED AND OPENED THIS QGIS PROJECT AND THE ERROR STILL APPEARS, PLEASE HELP ME, THANK YOU.

see the screenshots of the images
https://ibb.co/ThkZZ5x
https://ibb.co/94bJrKX
https://ibb.co/KrdNrd6](https://ibb.co/KrdNrd6)

**Comment by AndyPoes** [comment]
I'm having the same problem. Please help! My remotior sensus is up to date.

**Comment by AndyPoes** [comment]
I solved it by installing an older version of the plugin. Just select download from ZIP in the plugin tab. https://plugins.qgis.org/plugins/SemiAutomaticClassificationPlugin/version/7.10.11/

**Comment by semiautomaticgit** [comment]
Hello @macawills , 

> Hello dear developer community, I'm having a problem with the scp plugin due to the following message: IT SAYS THAT THE SENSUS REMOTE IS MISSING TO UPDATE, I CLOSED AND OPENED THIS QGIS PROJECT AND THE ERROR STILL APPEARS, PLEASE HELP ME, THANK YOU.

This is not an error, but a warning message. It indicates that the dependency Remotior Sensus is outdated and should be updated.
Updating the library depends on the way you installed the SCP.

In suggest to uninstall the plugin and QGIS, then reinstall the latest version of QGIS.
Finally install the SCP by following this guide
https://semiautomaticclassificationmanual.readthedocs.io/en/latest/installation.html 

In case of difficulties in installing Python dependencies you can follow this alternative method https://semiautomaticclassificationmanual.readthedocs.io/en/latest/faq.html#plugin-installation-4

**Comment by mong00se007** [comment]
The solution for me was to install the plugin: QPIP that will update the dependency.
